Cargo.toml

```diff
@@ -11,8 +11,8 @@ keywords = ["stdout", "stderr", "stdio", "redirect"]
 license = "MIT"
 
 [dependencies]
-libc = "0.2"
 tempfile = "3.0"
+filedescriptor = { version = "0.7.3", git = "https://github.com/robinbernon/wezterm", branch = "AddingStdRedirects" }
```

Comment:
I'll change this to a fixed version number once my PR for filedescriptor gets through.

Reply:
My PR went through. Filedescriptor is now using a public version number.